### PR TITLE
[darwin-framework-tool] [darwin-framework-tool] Add an optional parameter to the storage view command to only show the entries for a given commissioner

### PR DESCRIPTION
**Pull request by vivien-apple**
#### Problem

This PR adds an option to filter by controller (`identity [alpha, beta, gamma]`) when inspecting the contents of per-controller storage, making it easier to navigate. It also improves consistency in the use of `fabricId` alongside `controllerId`.

### CODE REVIEW

**Review by kiel-apple — APPROVED** [review]

File: examples/darwin-framework-tool/commands/common/ControllerStorage.mm [hunk 1]
```diff
@@ -143,6 +143,20 @@ - (void)storeValue:(NSData *)value forKey:key
...
+- (void)print
+{
+    NSLog(@"%@ (%@)", kDarwinFrameworkToolControllerDomain, _keyScopingPrefix);
+    for (NSString * controllerKey in self.storage) {
+        if (![self _isControllerScopedKey:controllerKey]) {
+            continue;
+        }
+
+        __auto_type * key = [self _controllerScopedKeyToKey:controllerKey];
+        __auto_type * data = [self.storage objectForKeyedSubscript:controllerKey];
```
Comment (bzbarsky-apple) [inline examples/darwin-framework-tool/commands/common/ControllerStorage.mm:155]: Shouldn't that just use `objectForKey`?

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/darwin-framework-tool/commands/common/ControllerStorage.mm (hunk 1 above)
Comment (vivien-apple) [reply examples/darwin-framework-tool/commands/common/ControllerStorage.mm:155]: > Shouldn't that just use `objectForKey`?

No. But that is true that I can simplify it: `__auto_type * data = self.storage[controllerKey]`

Comment (vivien-apple) [reply examples/darwin-framework-tool/commands/common/ControllerStorage.mm]: I have pushed an updated version.

**Comment by bzbarsky-apple** [comment]
Fast-tracking platform-specific fix with platform owner review.

**Pull request merged**